test/e2e/util.go

```diff
@@ -333,7 +341,7 @@ func kubectlCmd(args ...string) *exec.Cmd {
 	kubectlArgs := append(defaultArgs, args...)
 
 	//TODO: the "kubectl" path string might be worth externalizing into an (optional) ginko arg.
-	cmd := exec.Command("kubectl", kubectlArgs...)
+	cmd := exec.Command(filepath.Join(testContext.RepoRoot, "cluster/kubectl.sh"), kubectlArgs...)
```

Comment:
SGTM. That will let us use the latest build kubectl for gke-ci and the gcloud-bundled one for gke-test|staging|prod.